### DISCUSSION TITLE
What SQL statement is sent to Sqlalchemy when pool_pre_ping=True

### DISCUSSION THREAD

**Discussion by sollymay** [psycopg/psycopg2#1694] [Q&A] [answered]
I've been trying to read through the documentation to understand if psycopg2 sends anything other than a `SELECT 1` when the `pool_pre_ping` value is set to `True` on sqlalchemy or if this is the default?

**Comment by dvarrazzo** [comment, marked as answer]
You should ask the SQLAlchemy project not us...

Paycopg2 doesn't support sending an empty query, so likely they send a SELECT 1, but you should ask them.
